api/.gitignore

```diff
@@ -1,48 +1,40 @@
-bin
-obj
-csx
-.vs
-edge
-Publish
-
-*.user
-*.suo
-*.cscfg
-*.Cache
-project.lock.json
-
-/packages
-/TestResults
-
-/tools/NuGet.exe
-/App_Data
-/secrets
-/data
-.secrets
-appsettings.json
-local.settings.json
```

Comment:
Do we want this file checked in to the codebase? I have it locally and it will be checked in next commit I do unless I `rm` it.

Reply:
No, we shouldn't check it in to the codebase.